### ISSUE TITLE
Tag release in 6.x tree with ringphp and streams dependencies fixed

### ISSUE THREAD

**Issue by dinamic** [elastic/elasticsearch-php#1020]
```
Package guzzlehttp/ringphp is abandoned, you should avoid using it. No replacement was suggested.
Package guzzlehttp/streams is abandoned, you should avoid using it. No replacement was suggested.
```

I know this issue has been fixed with elasticsearch 7.x, but how about 6.x? We're not ready to upgrade yet.

Would it be possible to backport the removal of `guzzlehttp/ringphp` and `guzzlehttp/streams` to the 6.x tree?

**Comment by ezimuel** [comment]
@dinamic good point. I already planned to work on a 6.8 release starting next week. The goal is to include also the XPack endpoints, as I did for [7.7](https://github.com/elastic/elasticsearch-php/tree/7.7) coming release. I will also update the ringphp dependencies.

**Comment by anilcancakir** [comment]
+1 in 7.7

**Comment by olsgreen** [comment]
What's the status of this? Do you still intend to implement it?

**Comment by tyteen4a03** [comment]
Bump

**Comment by ezimuel** [comment]
Just released [6.8.0](https://github.com/elastic/elasticsearch-php/releases/tag/v6.8.0) that includes the fix for this issue.
